Story/Fireisland/00DoAllFireisland.cs

```diff
@@ -0,0 +1,21 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+//cs_include Scripts/Story/Fireisland/CoreFireisland.cs
+
+using Skua.Core.Interfaces;
+
+public class DoAllFireisland
+{
+    public IScriptInterface Bot => IScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+    public CoreStory Story = new();
```

Comment:
Dont need the public CoreStory line for Fireisland more laguna either less its in the public/void